docs/configuration-reference.md

```diff
@@ -2037,3 +2040,148 @@ exe:
 cmdline:
   [- <string>]
 ```
+
+### mysqld_exporter_config
+
+The `mysqld_exporter_config` block configures the `mysqld_exporter` integration,
+which is an embedded version of
+[`mysqld_exporter`](https://github.com/prometheus/mysqld_exporter)
+and allows for collection metrics from MySQL servers.
+
+Note that currently, an Agent can only collect metrics from a single MySQL
+server. If you want to collect metrics from multiple servers, run multiple
+Agents and add labels using `relabel_configs` to differentiate between the MySQL
+servers:
+
+```yaml
+mysqld_exporter:
+  enabled: true
+  data_source_name: root@(server-a:3306)/
+  relabel_configs:
+  - source_labels: [__address__]
+    target_label: mysql_server
```

Comment:
Does the mixin expect this label and does it use it as a variable? Typically this would be `instance`, right?

Reply:
I picked a new label because if the integrations subsystem is configured with `replace_instance_label` (which is the default), the instance label is always replaced with the IP of the host running the Agent. That applies for all integrations and the value can't be overwritten. 

You make a good point though, it'd be better to treat that behavior as a bug and have this use `instance` instead. I've opened up #220 to track this, and I'll change the example here to replace `instance` in the interrim. 

I'll fix #220 before putting out the release with both of our exporters.